ci/terraform/delivery-receipts/shared.tf

```diff
@@ -14,4 +14,7 @@ locals {
   bulk_user_email_table_encryption_key_arn = data.terraform_remote_state.shared.outputs.bulk_user_email_table_encryption_key_arn
   user_profile_encryption_policy_arn       = data.terraform_remote_state.shared.outputs.user_profile_encryption_policy_arn
   user_profile_kms_key_arn                 = data.terraform_remote_state.shared.outputs.user_profile_kms_key_arn
+
+  slack_event_sns_topic_arn = data.terraform_remote_state.shared.outputs.slack_event_sns_topic_arn
```

Comment:
As previous, potentially the same missing `dynatrace_secret` here too.

Fairly confident I'm missing something in my understanding of terraform locals.

Reply:
https://github.com/govuk-one-login/authentication-api/pull/5644#discussion_r1903942594